<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-25SBVBQ" failures="0" tests="4" name="Automation.profileon.PimTest" time="0.015" errors="1" timestamp="2022-12-11T11:43:12 IST" skipped="3">
  <testcase classname="Automation.profileon.PimTest" name="initialize" time="0.015">
    <error message="C:\Users\Admin\Java-Selenium\profileon\src\main\java\resources\data.properties (The system cannot find the path specified)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: C:\Users\Admin\Java-Selenium\profileon\src\main\java\resources\data.properties (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:158)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112)
at resources.base.initializeDriver(base.java:23)
at Automation.profileon.PimTest.initialize(PimTest.java:26)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:643)
at org.testng.TestRunner.beforeRun(TestRunner.java:632)
at org.testng.TestRunner.run(TestRunner.java:594)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- initialize -->
  <system-out/>
  <testcase classname="Automation.profileon.PimTest" name="SrchEmployeeByNameIdStatus" time="0.000">
    <skipped/>
  </testcase> <!-- SrchEmployeeByNameIdStatus -->
  <system-out/>
  <testcase classname="Automation.profileon.PimTest" name="SearchEmployeeByNameAndId">
    <skipped/>
  </testcase> <!-- SearchEmployeeByNameAndId -->
  <system-out/>
  <testcase classname="Automation.profileon.PimTest" name="SearchEmployeeByName">
    <skipped/>
  </testcase> <!-- SearchEmployeeByName -->
  <system-out/>
</testsuite> <!-- Automation.profileon.PimTest -->
